modules/accounts/accounts.js

```diff
@@ -1,191 +1,250 @@
+            Logger.warn(`Incorrect usage of activate-testnet-account command \nCorrect usage: - tezster activate-testnet-account <account-label>`);
             return;
         }
-        await this.loadTezsterConfig(); 
         
-        let result = await this.activateAlphanetAccount(args[0]);
-        console.log(result);
-        console.log(Helper.outputInfo(`If this account has already been activated, it may throw 'invalid_activation' error. You can visit https://${TESTNET_NAME}.tzstats.com for more information on this account`));
+        await this.activateAlphanetAccount(args[0]);
+        Logger.warn(`If this account has already been activated, it may throw 'invalid_activation' error. You can visit https://${TESTNET_NAME}.tzstats.com for more information on this account`);
+    }
+
+    async removeAccount(args) {
+        Logger.verbose(`Command : tezster remove-account ${args}`);
+        if (args.length < 1) {
+            Logger.warn(`Incorrect usage of remove-account command \nCorrect usage: - tezster remove-account <account-label/identity/hash>`);
+            return;
+        }
+        await this.deleteAccount(args[0]);
     }
 
     setProviderAccounts(args){    
         config.provider = args[0];
         jsonfile.writeFile(confFile, config);
-        eztz.node.setProvider(config.provider);
-        return Helper.outputInfo('Provider updated to ' + config.provider);
+        return Logger.info('Provider updated to ' + config.provider);
     }
 
     getProviderAccounts(){    
         if (config.provider) {
-            return Helper.outputInfo(config.provider);
+            return Logger.info(config.provider);
         } else {
-            return Helper.outputInfo('No provider is set');
+            return Logger.warn('No provider is set');
         } 
     }
 
-    getBalanceAccounts(account) {
+    async getBalanceAccounts(account) {
         var pkh = account, f;
+        const tezosNode = config.provider;
         if (f = Helper.findKeyObj(config.identities, pkh)) {
             pkh = f.pkh;
         } else if (f = Helper.findKeyObj(config.accounts, pkh)) {
             pkh = f.pkh;
         } else if (f = Helper.findKeyObj(config.contracts, pkh)) {
             pkh = f.pkh;
         }
-        return eztz.rpc.getBalance(pkh).then(function(r){
-            return Helper.output(Helper.formatTez(r/100));
-        }).catch(function(e){
-            return Helper.outputError(e);
+
+        const keys = this.getKeys(account);
+        if(!keys) {
+            return Logger.error(`Account with this label doesn't exists.`);
+        }
+
+        await RpcRequest.fetchBalance(tezosNode, pkh).then(function(body){
+            var balance = JSON.parse(body);
+            Logger.info(Helper.formatTez(balance));    
+        }).catch(function (error) {
+            Logger.error(`${error}`);
```

Comment:
https://itnext.io/javascript-promises-and-async-await-as-fast-as-possible-d7c8c8ff0abc
https://javascript.info/async-await

Reply:
Please fix this. This is not proper way to use async/await.
you should not use then() and catch() with await.